_shared_content/automate/triggers.md

```diff
@@ -24,3 +24,31 @@ This is a `Block` button, such as the one you will find hereunder, for the examp
 ### Cron
 
 The `Cron` trigger allows a user to periodically launch an automatic action, that should be defined and created by a user.
+
+## Logs and errors in triggers
+
+Some configuration errors in triggers can be hard to understand. We display all logs so it's easier to understand what can be improved and why the playbook is not working properly.
+
+### Logs
+
+In the playbooks listing page, you can identify a playbook with configuration errors via an error message in the playbook card.
+In a playbook page with configuration errors in a trigger node, you can see if the state of the node is in error.
+Click on this node and go to the tab “Errors” or “Logs” to see all messages.
+
+### Levels and retention
+
+There are five possible types of log messages :
+* `Debug` : used to find the reason in case your program is not working as expected or an exception has occurred
```

Comment:
```suggestion
There are five possible types of log messages :

* `Debug` : used to find the reason in case your program is not working as expected or an exception has occurred
```